modules/client/examples/index.ts

```diff
@@ -46,16 +46,24 @@ export const contextParams: ContextParams = {
   signer: new Wallet("private-key"),
   // Optional on "rinkeby", "arbitrum-rinkeby" or "mumbai"
   // Pass the address of the  `DaoFactory` contract you want to use. You can find it here based on your chain of choice: https://github.com/aragon/core/blob/develop/active_contracts.json
+  // By default we will use the one in the ethers-contract package
   daoFactoryAddress: "0x1234381072385710239847120734123847123",
+  // Pass the address of the ensRegistry, if its not provided, for mainnet and worli we will use the defualt ones
+  // For polygon and mumbai we will use the ones deployed by ourselves, you can check them here: https://github.com/aragon/core/blob/develop/active_contracts.json
+  ensRegistryAddress: "0x1234381072385710239847120734123847123",
```

Comment:
```suggestion
  // Optional. Pass the address of the ensRegistry for networks other than Mainnet or Goerli. 
  // It will default to the registry deployed by Aragon. You can check them here: https://github.com/aragon/osx/blob/develop/active_contracts.json
  ensRegistryAddress: "0x1234381072385710239847120734123847123",
```